client/src/components/TagBadge/TagBadge.component.tsx

```diff
@@ -0,0 +1,50 @@
+import { Link } from 'react-router-dom'
+import { getRedirectURL } from '../../util/urlparser'
+import './TagBadge.styles.scss'
+import { useGoogleAnalytics } from '../../contexts/googleAnalytics'
+import * as FullStory from '@fullstory/browser'
+import { TagType } from '~shared/types/base'
+import { Agency } from '../../services/AgencyService'
+
+const TagBadge = ({
+  tagName,
+  tagType,
+  agency,
+}: {
+  tagName: string
+  tagType: TagType
+  agency: Agency
+}): JSX.Element => {
+  const TagComponent = <div className="tag-badge">{tagName.toUpperCase()}</div>
+
+  const googleAnalytics = useGoogleAnalytics()
+
+  const sendClickTagEventToAnalytics = () => {
+    const timeToTagClick = new Date().getTime() - googleAnalytics.appLoadTime
+    googleAnalytics.sendUserEvent(
+      googleAnalytics.GA_USER_EVENTS.CLICK_TAG,
+      tagName,
+      timeToTagClick,
+    )
+    googleAnalytics.sendTiming(
+      'User',
```

Comment:
noticed we are slotting this into the same category as `sendUserEvent` - would providing a different category make the GA console more organised? up to you, I'm not super familiar with this

Reply:
I think its fine because we can search by the event label on GA? Either way should be fine, but User is a pretty big category so I just assumed it belonged there.